src/aks-preview/azcli-aks-live-test/az_aks_tool/ext.py

```diff
@@ -0,0 +1,88 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import glob
+import os
+from azdev.operations.testtool import _discover_module_tests
+from azdev.utilities import EXTENSION_PREFIX, get_path_table, get_name_index
+
+import utils
+
+
+def get_ext_test_index(mod_name):
+    # path table & name index
+    path_table = get_path_table()
```

Comment:
what's path table?

Reply:
key value pairs of all modules(in azcli & extention) and absolute path, used later to find test indexes

Reply:
> key value pairs of all modules(in azcli & extention) and absolute path, used later to find test indexes

then maybe put your comments"key value pairs of all modules(in azcli & extention) and absolute path, used later to find test indexes"  in the code?

Reply:
sure, fixed in commit [917128c](https://github.com/Azure/azure-cli-extensions/pull/3292/commits/917128c2bffd73b5cc6ab77a40253ea2f4914218).